src/Sulu/Component/SmartContent/Orm/DataProviderRepositoryTrait.php

```diff
@@ -60,6 +60,7 @@ private function findByFiltersIds($filters, $page, $pageSize, $limit, $locale, $
 
         $queryBuilder = $this->createQueryBuilder('c')
             ->select('c.id')
+            ->distinct()
```

Comment:
@chirimoya This was the easiest solution we could find. Another solution would be `groupBy('id')`. I have read that this might be faster in some situations, but I've used an `EXPLAIN` statement to check the execution plan, and they were identical for both solutions. So I decided to go with `distinct`, because it explains better what is actually happening here.